src/category_theory/abelian/exact.lean

```diff
@@ -191,6 +191,61 @@ lemma epi_iff_cokernel_π_eq_zero : epi f ↔ cokernel.π f = 0 :=
 
 end
 
+section opposite
+
+instance exact.op [exact f g] : exact g.op f.op :=
+begin
+  rw exact_iff,
+  refine ⟨by simp [← op_comp], _⟩,
+  apply_fun quiver.hom.unop using quiver.hom.unop_inj,
+  simp only [unop_comp, cokernel.π_op, eq_to_hom_refl, kernel.ι_op, category.id_comp,
+    category.assoc, kernel_comp_cokernel_assoc, zero_comp, comp_zero, unop_zero],
+end
+
+lemma exact.op_iff : exact f g ↔ exact g.op f.op :=
+⟨λ e, @@exact.op _ _ _ _ e, λ e, begin
+  rw exact_iff at e ⊢,
+  refine ⟨by convert (congr_arg quiver.hom.unop e.1), _⟩,
+  have e3 := ((cokernel_op_unop g).inv ≫= congr_arg quiver.hom.unop e.2) =≫ (kernel_op_unop f).hom,
+  simp only [← category.assoc, iso.inv_hom_id, unop_comp, cokernel.π_op, kernel.ι_op,
+    category.id_comp, eq_to_hom_refl, category.comp_id, unop_zero, comp_zero, zero_comp] at e3,
+  simpa only [category.assoc, iso.inv_hom_id, category.comp_id] using e3,
+end⟩
+
+
+instance exact.unop {X Y Z : Cᵒᵖ} (g : X ⟶ Y) (f : Y ⟶ Z) [exact g f] : exact f.unop g.unop :=
+begin
+  have e0 : exact g f := infer_instance,
+  rw abelian.exact_iff at e0 ⊢,
+  have e02 := e0.2,
+  refine ⟨by convert (congr_arg quiver.hom.unop e0.1), _⟩,
+  have eq2 : (kernel.ι f).unop = _ := @kernel.ι_op C _ _ Z.unop Y.unop f.unop,
+  have eq1 : (cokernel.π g).unop = _ := @cokernel.π_op C _ _ Y.unop X.unop g.unop,
+  apply_fun quiver.hom.unop at e02,
+  rw [unop_comp, unop_zero, eq1, cokernel_op_unop_hom, ← category.assoc, limits.kernel.lift_ι,
+    eq_to_hom_refl, category.comp_id, cokernel.π_op, eq_to_hom_refl, category.comp_id, eq2,
+    eq_to_hom_refl, category.id_comp] at e02,
+  have e03 := (cokernel_op_unop g.unop).inv ≫= (e02 =≫ (kernel_op_unop f.unop).hom),
+  rwa [← category.assoc, ← category.assoc, ← category.assoc, ← category.assoc, iso.inv_hom_id,
+    category.id_comp, category.assoc, iso.inv_hom_id, category.comp_id, zero_comp, comp_zero]at e03,
+end
```

Comment:
Once you have `exact.op_iff`, you can prove this very easily as follows:
```lean
instance exact.unop {X Y Z : Cᵒᵖ} (g : X ⟶ Y) (f : Y ⟶ Z) [h : exact g f] : exact f.unop g.unop :=
begin
  rw [← f.op_unop, ← g.op_unop] at h, 
  rwa exact.op_iff,
end
```

`exact.unop_iff` can be golfed to 1 or 2 lines in a similar way.